packages/gitgraph-core/src/gitgraph.ts

```diff
@@ -38,6 +39,7 @@ interface GitgraphOptions {
   author?: string;
   commitMessage?: string;
   generateCommitHash?: () => Commit["hash"];
+  branchesOrderFunction?: BranchesOrderFunction;
```

Comment:
As said previously, I'll merge the code since it's good, but I'll probably rename this `sortBranches` before publishing.